scripts/patchPreview.js

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const fs = require('fs');
 
-const args = process.argv.slice(2);
-if (!args || args.length == 0) {
-  throw new Error('No version parameter was provided.');
-}
+const date = new Date(new Date().toLocaleString('en-US', { timeZone: 'America/New_York' }));
```

Comment:
Why EST?

Reply:
We're Boston based, thus this time zone :) I don't have any strong preference though 😄